src/main/java/com/example/onlinebookstore/config/MapperConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.example.onlinebookstore.config;
+
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.NullValueCheckStrategy;
+
+@org.mapstruct.MapperConfig(
```

Comment:
you can import this annotation in the import statement.
```suggestion
@MapperConfig(
```

Reply:
How? i can't write this annotation without path